### PR TITLE
[FrontendV2]Fix for item on every navbar page is floating towards right in  for screen width more than 1440px 

### LINKED ISSUE #3195: [FrontendV2] Item on every navbar page is floating towards right in  for screen width more than 1440px
Frontend v2 is unoptimized for screen width greater than 1440px.

Item on Participant challenges navbar page is floating towards right in  for screen width more than 1440px, Same behaviour can be seen under in all navbar page section.

Please look at following image for better understanding.
Screenshot from 2021-02-01 21-22-09

Frontend v2 is unoptimized for screen width greater than 1440px

### PR DESCRIPTION
**Pull request by Alabhya268**
This PR fixes #3195 and #3241.

Before -

106642510-6fd6e000-65ae-11eb-89f4-aef17d3ac9b6

After -

Screenshot from 2021-02-04 13-33-37

### CODE REVIEW

**Comment by Alabhya268** [comment]
@RishabhJain2018 @Ram81 , have a look at this PR.

**Comment by Alabhya268** [comment]
@RishabhJain2018 , In [CONTRIBUTING.md](https://github.com/Alabhya268/EvalAI/blob/master/.github/CONTRIBUTING.md) rather than using  "git merge upstream/master" we should use "git pull upstream master" .

As "git merge upstream/master" causes also added unrelated commit history as it can be seen in this PR that commit made by you are also showing.

Should I make a PR to repalce "git merge upstream/master" with "git pull upstream master" in CONTRIBUTING.md ?

**Review by Ram81 — DISMISSED** [review]
.

File: frontend_v2/src/styles/base.scss [hunk 1]
```diff
@@ -2154,13 +2154,10 @@ tr {
     zoom: 1 !important;
   }
 
-  .web-container {
```
Comment (Ram81) [inline frontend_v2/src/styles/base.scss:2157]: @Alabhya268 why are we removing this css class? Are you sure it is not messing up css style for other pages?

**Review by Ram81 — CHANGES_REQUESTED** [review]

File: frontend_v2/src/styles/base.scss (hunk 1 above)
Comment (Alabhya268) [reply frontend_v2/src/styles/base.scss:2157]: @Ram81 , I sure it's not having any negative effects on css style. I probably can find other way around but that seems unecessary. Let me know what you think.

**Comment by Alabhya268** [comment]
@Ram81 , Though removing web-container didn't had any negative effect , I have added web-container again with width: unset,
Hope this should lessen your concern.

**Comment by Alabhya268** [comment]
@Ram81 , Please review this PR again.

**Review by Ram81 — APPROVED** [review]
LGTM

**Review by RishabhJain2018 — APPROVED** [review]

**Comment by Alabhya268** [comment]
@RishabhJain2018 could you please merge this PR so that I can work on #3243 .

**Pull request merged**